src/js/components/DataTableColumns/__tests__/DataTableColumns-test.tsx

```diff
@@ -38,6 +39,34 @@ describe('DataTableColumns', () => {
 
     expect(getByRole('button', { name: 'Open column selector' })).toBeTruthy();
     expect(container.firstChild).toMatchSnapshot();
```

Comment:
While we're here, should we update these to `asFragment()` instead of `container.firstChild`?

Reply:
we can yeah